src/Components/NotFoundErrorPage.tsx

```diff
@@ -102,8 +102,8 @@ const NotFound = connect(function NotFound() {
   )
 })
 
-function hasPortalAppContent(): boolean {
-  return !!Obj.onAllSites().get(import.meta.env.SCRIVITO_ROOT_OBJ_ID)
+function missingPortalAppContent(): boolean {
```

Comment:
* [ ] FYI Not sure about our naming conventions, but we have some boolean functions starting with `is`. Maybe we should name the two helpers in this module with `is`/`has` prefixes for consistency.